debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/converters/sink/SinkConverter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.converters.sink;
+
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.bson.BsonDocument;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.connector.mongodb.converters.sink.LazyBsonDocument.Type;
+import io.debezium.table.ColumnNamingStrategy;
+
+public class SinkConverter {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SinkConverter.class);
+
+    private static final SinkRecordConverter SCHEMA_RECORD_CONVERTER = new SchemaRecordConverter();
+    private static final SinkRecordConverter MAP_RECORD_CONVERTER = new MapRecordConverter();
+    private static final SinkRecordConverter JSON_STRING_RECORD_CONVERTER = new JsonStringRecordConverter();
+    private static final SinkRecordConverter BYTE_ARRAY_RECORD_CONVERTER = new ByteArrayRecordConverter();
+
+    public SinkDocument convert(final SinkRecord record, ColumnNamingStrategy columnNamingStrategy) {
+        LOGGER.debug("record: {}", record);
```

Comment:
This class makes use of debug logging but similar ones use trace. Can we pick one and be consistent?

Reply:
Please use `io.debezium.util.Loggings` whenever the data could be printed to the log.

Reply:
@Naros  This is intended. While `debug` level gives you good debug logging info the `trace` level gives you very detailed additional info to figure out where in the code what was happening. For general error finding `debug` will be good enough level and I want to keep much more details on the `trace` level which is used only in the data type conversion to get much more details on every processing step. It helped me a lot on the data type mapping change 3 weeks ago.

@jpechane is this mandatory for normal log levels or for debug/trace level too?

Reply:
@rk3rn3r For security reasons we don't log the content of messages in the level other than trace. This came multiple time as a security concern from different community members.

Reply:
Understood! done